pipeline/backend/kubernetes/pod.go

```diff
@@ -136,6 +150,7 @@ func Pod(namespace string, step *types.Step, labels, annotations map[string]stri
 			RestartPolicy:      v1.RestartPolicyNever,
 			HostAliases:        hostAliases,
 			NodeSelector:       nodeSelector,
+			Tolerations:        tolerations,
```

Comment:
And maybe logging (trace) actual values which pod is going to run with is good idea (see node selector logging).

Reply:
Can you check the implementatiion?

Reply:
Yeah, can test it tomorrow.